coreapi/main_api.py

```diff
@@ -564,8 +564,8 @@ def similarface(request, filename):
     if not os.path.exists(file_folder):
         os.makedirs(file_folder)
 
-    handle_uploaded_file(request.FILES['file'], file_folder + '/referenceImage.jpg')
-    handle_uploaded_file(request.FILES['compareImage'], file_folder + '/compareImage.jpg')
+    handle_uploaded_file(reference_img, file_folder + '/referenceImage.jpg')
```

Comment:
use os.path.join to join the directory and filename

Reply:
done